<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoForm</class>
 <widget class="QWidget" name="InfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>231</width>
     <height>349</height>
    </rect>
   </property>
   <property name="title">
    <string>Действие</string>
   </property>
   <widget class="QPushButton" name="showPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Просмотреть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitToMainPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть вкладку</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>701</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Пункт назначения</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>681</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>Поиск рейсов по пункту назначения</string>
     </property>
     <widget class="QLineEdit" name="destinationLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Пункт назначения</string>
      </property>
     </widget>
     <widget class="QTableView" name="destinationTableView">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>471</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Время отправления</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>681</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>Поиск рейсов по времени отправления</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Начальное время</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="firstTimeEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="lastTimeEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Конечное время</string>
      </property>
     </widget>
     <widget class="QTableView" name="timeTableView">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>471</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
